pkg/agent/cniserver/pod_configuration.go

```diff
@@ -416,7 +416,14 @@ func (pc *podConfigurator) reconcile(pods []corev1.Pod, containerAccess *contain
 		desiredPods.Insert(k8s.NamespacedName(pod.Namespace, pod.Name))
 	}
 
+	missingIfConfigs := make([]*interfacestore.InterfaceConfig, 0)
 	for _, containerConfig := range knownInterfaces {
+		// Find the OVS ports which are not connected to host interfaces, this is useful on Windows if the runtime is
```

Comment:
, this -> . This

Reply:
Could we add some comments somewhere (pod_configuration.go, or pod_configuration_windows.go) to briefly describe the process of interface creation on Windows with Docker and containerd? It can be very short, but at least for readers to understand the CNI call sequence and what happen at each call.

Reply:
Sure, I would update.